src/common.cpp

```diff
@@ -1,13 +1,12 @@
 #include "common.hpp"
-
```

Comment:
Please restore this empty line.

That helps the reader understand that this cpp file is the implementation of the first header file.